### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by kirillseva**
Reduced CI build time from 4-7 minutes to 1.5 minutes and added acceptance tests using `chai` and `mocha`.

@robertzk Discovered some json serialization bugs in responses, we should fix them in a follow-up PR

### CODE REVIEW

**Comment by peterhurford** [comment]
Nice idea 👍

**Comment by kirillseva** [comment]
🙇

File: README.md [hunk 1]
```diff
@@ -5,14 +5,30 @@ Minimal R server mimicking Ruby's Sinatra gem.
 
 # Installation
 
-This package is not yet available from CRAN (as of October 5, 2015).
+This package is not yet available from CRAN (as of July 5, 2016).
```
Comment (robertzk) [inline README.md:8]: Thanks :)

File: inst/simple_server.R [hunk 2]
```diff
@@ -0,0 +1,6 @@
+library(methods)
+devtools::load_all()
+
+routes <- source(microserver:::packagefile('inst/routes.R'))$value
```
Comment (robertzk) [inline inst/simple_server.R:4]: Apparently `dget` is a synonym for `source(...)$value`

File: test/smoke.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      it('can parse query parameters', function(done) {
+        var query = { name: 'foo', limit: 1 }
+        chai.request(base_url)
+          .get('/parse_query')
+          .query(query)
+          .end(function(err, res) {
+            should.not.exist(err);
+            res.should.have.status(200);
+            res.body.query.name.should.equal(query.name);
+            console.log(res.body);
```
Comment (robertzk) [inline test/smoke.js:63]: rm?

Comment (kirillseva) [inline test/smoke.js:63]: yeah 👍

File: inst/simple_server.R (hunk 2 above)
Comment (kirillseva) [inline inst/simple_server.R:4]: nice!

**Pull request merged**